backend/server3.js

```diff
@@ -0,0 +1,206 @@
+var express = require('express');
+var app = express();
+
+const bodyParser = require('body-parser');
+const { application } = require('express');
+
+var util = require('util');
+var encoder = new util.TextEncoder('utf-8');
+const { MongoClient } = require("mongodb");
+const { response } = require("express");
+const uri = "mongodb://127.0.0.1:27017"
+const client = new MongoClient(uri)
+
+//for accessing other apis
+const request = require('request');
+app.use(express.json());
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const OneSignal = require('@onesignal/node-onesignal') ;
+
+const app_key_provider = {
+    getToken() {
+        return 'MjJiMGQ3ODAtNjIzNC00ZjBkLTgxNGYtMDY0M2YxYWJlNjE0';
+    }
+};
+
+const configuration = OneSignal.createConfiguration({
+    authMethods: {
+        app_key: {
+        	tokenProvider: app_key_provider
+        }
+    }
+});
+const client3 = new OneSignal.DefaultApi(configuration);
+
+app.post('/sendToDevice3', async function(req, res){
```

Comment:
Codacy has a fix for the issue: [Replace `'/sendToDevice3',·async·function(req,·res)` with `"/sendToDevice3",·async·function·(req,·res)·`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion
app.post("/sendToDevice3", async function (req, res) {
```